assets/i18n/en_US.json

```diff
@@ -13,6 +13,7 @@
         "settingsTab": "Settings"
     },
     "homeView": {
+        "refreshSuccess": "Refresh successfully",
```

Comment:
Shouldn't this be "Successfully refresh"?

Reply:
Should either be
 - Successfully refreshed
 - Refreshed successfully

Reply:
do Refreshed successfully